cms/djangoapps/contentstore/views/course.py

```diff
@@ -1317,6 +1331,26 @@ def update_usage_info(store, course, configuration):
         configuration_json['usage'] = usage_information.get(configuration.id, [])
         return configuration_json
 
+    @staticmethod
+    def get_or_create_cohorted_user_partition(course, store, user_id):
+        """
+        Returns the first user partition from the course which uses the
+        CohortPartitionScheme, or generates one if no such partition is
+        found.  The created partition is not saved to the course until
+        the client explicitly creates a group within the partition and
+        POSTs back.
+        """
+        cohorted_partition = get_cohorted_user_partition(course.id)
+        if cohorted_partition is None:
+            cohorted_partition = UserPartition(
+                id=GroupConfiguration.generate_id(GroupConfiguration.get_used_ids(course)),
+                name='Cohort Group Configuration',  # TODO: confirm w/ doc/prod -- translate? (below as well)?
```

Comment:
The name is not shown anywhere (currently). I have a feeling "Content Group Configuration" would be a better name now. Don't think you need to translate this or description, since they are not shown.